core/src/processing/opengl/PSurfaceJOGL.java

```diff
@@ -195,7 +195,8 @@ protected void initGL() {
     if (profile == null) {
       if (PJOGL.profile == 2) {
         try {
-          profile = GLProfile.getGL2ES1();
+          // request at least GL2 or GLES2
+          profile = GLProfile.getGL2ES2();
```

Comment:
bumping the profile to GL2ES2 requires upgrading the shader code to [v330](https://github.com/mattdesl/lwjgl-basics/wiki/GLSL-Versions#version-330), at least on Mac OSX.